src/sentry/incidents/subscription_processor.py

```diff
@@ -106,6 +107,11 @@ def check_trigger_status(self, trigger, status):
         return incident_trigger is not None and incident_trigger.status == status.value
 
     def process_update(self, subscription_update):
+        if not features.has("organizations:incidents", self.subscription.project.organization):
```

Comment:
@EvanPurkhiser Were you saying that access to transaction/performance metrics are gated behind an additional feature? If that's the case, we might need to check the dataset of the query and then check that flag as well.

Reply:
Weird I thought I left a comment but maybe I didn't submit.

Yes, I think we do need to gate on whether they have `incidents-performance` and if it's a transaction datatype query. We'll also have to potentially hide incidents that are of the datatype as well.

Reply:
Ok, let's just add some simple logic here for now to check that extra feature flag if the dataset is transactions.